pages/learn/develop/local-mode.md

```diff
@@ -14,25 +14,25 @@ To use local mode on a device:
 - The device must be running a [development][development] variant of the OS.
 - You must have the [{{ $names.company.lower }} CLI][cli] installed on your development machine.
-- Local mode must be enabled through the {{ $names.cloud.lower }} dashboard. You can enable it from either the *Actions* menu of the device dashboard or click to expand the arrow located on the device dashboard and select *Enable Local Mode*.
+- Local mode must be enabled through the {{ $names.cloud.lower }} dashboard. You can enable it from the device _Settings_ tab.
 
-![Enable local mode](/img/local-mode/enable-local-mode.png)
+![Enable local mode](/img/local-mode/enable-local-mode-settings-tab.png)
 
 ## Local mode caveats
 
 - In local mode, a device will not send logs back to the {{ $names.cloud.lower }} dashboard. Refer to the [local mode logs section](#local-mode-logs) to view logs in local mode.
 - Device and service environment variables set from the {{ $names.cloud.lower }} will not be applied to local mode containers. It is still possible to set environment variables in your `docker-compose.yml` or `Dockerfile`.
 - Changes to device [configuration][configuration], for example, `BALENA_HOST_CONFIG_gpu_mem`, will result in the device rebooting and applying those settings.
 - Actions such as _Restart_ and _Purge data_ from the {{ $names.cloud.lower }} dashboard will not apply to local mode containers.
-- When switching out of local mode and back to tracking releases from {{ $names.cloud.lower }}, the Supervisor will destroy any local mode containers and  volumes, as well as clean up unneeded base images, and then start the release that {{ $names.cloud.lower }} instructs it to run.
+- When switching out of local mode and back to tracking releases from {{ $names.cloud.lower }}, the Supervisor will destroy any local mode containers and volumes, as well as clean up unneeded base images, and then start the release that {{ $names.cloud.lower }} instructs it to run.
 
 ![Device in local mode](/img/local-mode/device-in-local-mode-20-01-09.png)
 
 ## Scan the network and find your device
 
 Before you can get your app running on your device in local mode, you have to find your device. You can find the `short-uuid` and local IP address of the device from the device dashboard or by scanning the network. To perform a scan, login to the {{ $names.company.lower }} CLI and use `{{ $names.company.short }} scan` to find any local {{ $names.os.lower }} devices. All {{ $names.os.lower }} devices advertise themselves on the network using [Avahi][avahi]. The names take the form `<short-uuid>.local`, where the `short-uuid` is the UUID you see on your device dashboard.
 
-__Note:__ You may need administrator privileges to run `{{ $names.company.short }} scan` as it requires access to all network interfaces.
+**Note:** You may need administrator privileges to run `{{ $names.company.short }} scan` as it requires access to all network interfaces.
```

Comment:
If you refer to the page where this note section is present. You fill find a small yellow box that is being rendered when you specify a note using `__Note:__` that's why through out the docs they are referred to like this. 

Through user research we have concluded that these boxes divide a lot of our user's attention. So I do agree on converting a few of them to normal notes as you have done. 

If this wasn't your intention, then i recommend reverting this change because this will take away those yellow boxes being present. 

imo we should have less of them on a single page.